webpack/components/forms/FormEdit.js

```diff
@@ -167,9 +167,13 @@ class FormEdit extends Component {
     event.preventDefault();
     // Because of the way we have to pass the current questions in we have to manually sync props and state for submit
     let form = Object.assign({}, this.state);
+    let stats = Object.assign({}, this.props.stats);
     form.linkedQuestions = this.state.formQuestions;
     this.props.formSubmitter(form, (response) => {
       this.unsavedState = false;
+      stats.formCount = this.props.stats.formCount + 1;
```

Comment:
This does not check whether it is an edit or creating a new form. It will increment the form count even if you edit an existing form.